intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -402,9 +484,43 @@ \subsection{Connect}
 
 %% TODO: how to fix this on windows/putty? generally this is not an issue I believe
 \ifmacORlinux
+\ifantwerpen
 Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifbrussel
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifgent
+Open the .bash\_profile or the .profile on your local machine with your
+\fi
+\ifleuven
+Open the .bashrc or the .profile on your local machine with your
```

Comment:
It works. We just prefer our users to edit .bashrc - then we recognize those that modified anything...